### ISSUE TITLE
Release checklist

### ISSUE THREAD

**Issue by mattseddon** [iterative/vscode-dvc#429]
Things that need to happen before we make a public release announcement

- [x] Error handling and debugging - #470
- [x] Update Details section, changelog, get started -  https://github.com/iterative/vscode-dvc/issues/434, https://github.com/iterative/vscode-dvc/issues/435
- [x] Document that we collect telemetry information and users can opt-out by setting `telemetry.enableTelemetry` to false (https://github.com/iterative/vscode-dvc/issues/511#issuecomment-913986086).
- [x] Ask VS Code to prompt the user to install our extension when "dvc.yaml" or "dvc.lock" or ".dvc" file is present
- [x] Investigate [walkthroughs](https://code.visualstudio.com/api/references/contribution-points#contributes.walkthroughs)
- [x] Add plots into walkthrough
- [x] Check everything in walkthrough is up to date and correct (#1876)
- [x] Make repo public
- [x] Remove preview tag from extension (#1860)
- [x] README = marketplace listing -> the existing one needs to be reworked (#1816)
- [x] Add extra badges once the repo is public (https://shields.io/) (#1873)
- [x] Add repo banner (#1875)
- [x] Make sure VS Code is added to DVC README (https://github.com/iterative/dvc/pull/7859)
- [x] Add to iterative README (https://github.com/iterative)
- [x] [Verify Iterative as a publisher](https://code.visualstudio.com/updates/v1_62#_verified-extension-publishers)
- [x] Release pipeline (#754)
- [x] Publish to [Open VSX](https://open-vsx.org/) from the pipeline https://github.com/iterative/vscode-dvc/issues/1595
- [x] Internal alpha testing
- [x] Early adopter testing
- [x] Minification of code / Tree shaking (optional) - won't do
- [x] Implement light theme design (light svgs https://github.com/iterative/vscode-dvc/issues/175#issuecomment-876951701) - won't do
- [x] Consider extension pack ([Yaml parser](https://dvc.org/doc/install/plugins#visual-studio-code), vscode-cml and whatever the dvc-azure plugin is called) - consider later
- [x] Revisit Restricted Mode with a walkthrough explanation and/or allowing limited capability
- [x] Rename `master` to `main`
- [x] Automate changelogs
- [x] Create marketplace release checklist - won't do, after release has been further automate we can release more often instead
- [x] Check dependency licenses
- [x] Test against arbitrary large repo (https://github.com/iterative/vscode-dvc/issues/429#issuecomment-1082458061) - done for now

Won't do

- [ ] Rework of data management code after `dvc status` is released
- [ ] E2E tests
